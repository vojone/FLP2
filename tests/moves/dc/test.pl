:- consult('../../../cube').

:- read_cube(C, _), move_d_ccw(C, _, _, RC), write_cube(RC).
