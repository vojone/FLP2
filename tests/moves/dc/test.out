555
555
555
444 111 222 333
111 222 333 444
222 333 444 111
369
258
147
